example/pages/index.vue

```diff
@@ -224,11 +224,11 @@ export default {
       bech32PrefixConsPub: 'likevalconspub',
       availableMethods: [
         LikeCoinWalletConnectorMethodType.Keplr,
+        [LikeCoinWalletConnectorMethodType.KeplrMobile, { isRecommended: true }],
```

Comment:
Should we also push the method to tier 1 if recommend is set?

Reply:
Users can config which tier they want

Added example usage